iothub/service/src/DigitalTwin/DigitalTwinInvokeCommandRequestOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// General request options that are applicable, but optional, for invoke command APIs.
+    /// </summary>
+    public class DigitalTwinInvokeCommandRequestOptions
+    {
+        /// <summary>
+        /// The time (in seconds) that the service waits for the device to come online.
+        /// The default is 0 seconds (which means the device must already be online) and the maximum is 300 seconds.
+        /// </summary>
+        public int ConnectTimeoutInSeconds { get; set; }
```

Comment:
I think these should be nullable?

Reply:
The PL defines it as nullable, so I'll update the request options to be the same